### PR TITLE
Feature/91 create a parser to read the pathname

### PR DESCRIPTION
**Pull request by JWebCoder**

### CODE REVIEW

File: examples/vite-react/src/App.tsx [hunk 1]
```diff
@@ -2,22 +2,24 @@ import Lumina from '@lumina/core'
 import '@lumina/core/style.css'
 import { getFullData } from './lib/dataFetcher'
 import luminaConfig from './luminaComponents/config'
-import { BrowserRouter, Route, Routes } from 'react-router-dom'
+import { BrowserRouter, Route, Routes, useLocation } from 'react-router-dom'
+import { Contact } from './pages/contact'
```
Comment (bmirra01) [inline examples/vite-react/src/App.tsx:6]: are we keeping this dummy page for now?

File: examples/vite-react/src/pages/contact.tsx [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export const Contact = () => {
+  return <div>Contact page</div>
+}
```
Comment (bmirra01) [inline examples/vite-react/src/pages/contact.tsx:3]: Same as first comment

**Review by bmirra01 — CHANGES_REQUESTED** [review]
image

Not rendering anything on Vercel, this could be related to the issue I had with creating routing for the /editor. Check `vercel.json` on `examples/vite-react`, the solution most likely is there.

**Review by Felix-by-merkle — APPROVED** [review]

File: examples/vite-react/src/App.tsx (hunk 1 above)
Comment (JWebCoder) [reply examples/vite-react/src/App.tsx:6]: yes as an example of a page outside of Lumina

File: examples/vite-react/src/pages/contact.tsx (hunk 2 above)
Comment (JWebCoder) [reply examples/vite-react/src/pages/contact.tsx:3]: same answer :smile:

**Review by bmirra01 — APPROVED** [review]
All seems good then 👍 ready to merge

**Pull request merged**